ci/pipelines/deploy-to-test.yml

```diff
@@ -1452,6 +1467,36 @@ jobs:
     <<: *put_success_slack_notification
     <<: *put_failure_slack_notification
 
+  - name: record-toolbox-release-number
+    serial_groups: [hosted-graphite]
```

Comment:
I don't think there's any need for these to be in a serial group, feels like creating a bottleneck without a benefit. HG is expecting to receive vast quantities of metrics so it's not like we'll hit a rate limit etc

Reply:
Agreed, I was being overcautious when I first added in everything (in case they ALL failed! 🔥 ) Will remove